lib/stat-paths-for.js

```diff
@@ -1,3 +1,4 @@
+'use strict';
```

Comment:
@stefanpenner Note that I renamed `lib/stats-path-for.js → lib/stat-paths-for.js` because I think that's what you actually meant based off the jsdoc + function name. Correct me if I'm wrong.